### PR TITLE
Update translations and link

### PR DESCRIPTION
**Pull request by oterral**
Use new CMS links for disclaimer in share and draw.

[Test](https://mf-geoadmin3.dev.bgdi.ch/teo_trans/?lang=fr&topic=ech&bgLayer=ch.swisstopo.pixelkarte-farbe&X=190000.00&Y=660000.00&dev3d=true&zoom=1&debug&layers=ch.swisstopo.zeitreihen,ch.bfs.gebaeude_wohnungs_register,ch.bav.haltestellen-oev,ch.swisstopo.swisstlm3d-wanderwege&layers_visibility=false,false,false,false&layers_timestamp=18641231,,,)

### CODE REVIEW

**Comment by loicgasser** [comment]
hum, you can close this one then:
https://github.com/geoadmin/mf-geoadmin3/pull/3422

File: src/locales/de.json [hunk 1]
```diff
@@ -66,7 +66,7 @@
 "draw_back": "Zurück / Zeichnen beenden",
 "draw_del_all_features": "Lösche alle Elemente ",
 "draw_del_selected_features": "Lösche ausgewählte Elemente ",
-"draw_delete": "Löschen",
+"draw_delete": "Delete",
```
Comment (loicgasser) [inline src/locales/de.json]: Is that wanted?

**Comment by oterral** [comment]
Arf sorry didn't see your PR

File: src/locales/de.json (hunk 1 above)
Comment (oterral) [inline src/locales/de.json]: Don't know :)

**Pull request merged**